reference/7/Microsoft.PowerShell.Utility/Test-Json.md

```diff
@@ -20,70 +20,89 @@ Test-Json [-Json] <string> [[-Schema] <string>] [<CommonParameters>]
+The Test-Json cmdlet tests whether a string is a valid JavaScript Object Notation (JSON) document
+and can optionally very that JSON document against a provided schema.
 
-Many web sites use JSON instead of XML to serialize data for communication between servers and web-based apps.
+The verified string can then be used with the `ConvertFrom-Json` cmdlet convert a JSON-formatted
+string to a JSON object, which is easily managed in PowerShell or sent to another program or web
+service that access JSON input.
 
-This cmdlet was introduced in Windows PowerShell 6.1
+Many web sites use JSON instead of XML to serialize data for communication between servers and
+web-based apps.
+
+This cmdlet was introduced in PowerShell 6.1
 
 ## EXAMPLES
 
 ### Example 1: Test if an object is valid JSON
+
+This example tests whether the input string is a valid JSON document.
+
+```powershell
+"{'name': 'Ashley', 'age': 25}" | Test-Json
 ```
-PS C:\> "{'name': 'Ashley', 'age': 25}" | Test-Json
+
+```Output
 True
 ```
-This command tests whether the input string is a valid JSON document, returning True since the string is valid JSON.
 
 ### Example 2: Test an object against a provided schema
+
+This example takes a string containing a JSON schema and compares it to an input string. Since the
+input string conforms to the schema described in the **Schema** parameter the cmdlet returns
+`$True`.
+
 ```powershell
-PS C:\> $schema = '{
->>   "definitions": {},
->>   "$schema": "http://json-schema.org/draft-07/schema#",
->>   "$id": "http://example.com/root.json",
->>   "type": "object",
->>   "title": "The Root Schema",
->>   "required": [
->>     "name",
->>     "age"
->>   ],
->>   "properties": {
->>     "name": {
->>       "$id": "#/properties/name",
->>       "type": "string",
->>       "title": "The Name Schema",
->>       "default": "",
->>       "examples": [
->>         "Ashley"
->>       ],
->>       "pattern": "^(.*)$"
->>     },
->>     "age": {
->>       "$id": "#/properties/age",
->>       "type": "integer",
->>       "title": "The Age Schema",
->>       "default": 0,
->>       "examples": [
->>         25
->>       ]
->>     }
->>   }
->> }'
-PS C:\> "{'name': 'Ashley', 'age': 25}" | Test-Json -schema $schema
-True
+$schema = @'
+{
+  "definitions": {},
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://example.com/root.json",
+  "type": "object",
+  "title": "The Root Schema",
+  "required": [
+    "name",
+    "age"
+  ],
+  "properties": {
+    "name": {
+      "$id": "#/properties/name",
+      "type": "string",
+      "title": "The Name Schema",
+      "default": "",
+      "examples": [
+        "Ashley"
+      ],
+      "pattern": "^(.*)$"
+    },
+    "age": {
+      "$id": "#/properties/age",
+      "type": "integer",
+      "title": "The Age Schema",
+      "default": 0,
+      "examples": [
+        25
+      ]
+    }
+  }
+}
+'@
+"{'name': 'Ashley', 'age': 25}" | Test-Json -Schema $schema
```

Comment:
Would it be valuable to show if the JSON was `'age': '25'`, what the error would be (IntegerExpected: #/age) and not just a True result?

Reply:
fixed